addons/stock/migrations/13.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# Copyright 2020 ForgeFlow <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_copies = {
+    'stock_location': [
+        ('usage', None, None),
+    ],
+}
+
+_model_renames = [
+    ('report.stock.forecast', 'report.stock.quantity'),
+    ('stock.fixed.putaway.strat', 'stock.putaway.rule'),
+]
+
+_table_renames = [
+    ('stock_fixed_putaway_strat', 'stock_putaway_rule'),
+]
+
+_field_renames = [
+    ('stock.putaway.rule', 'stock_putaway_rule', 'fixed_location_id', 'location_out_id'),
+    ('stock.move', 'stock_move', 'propagate', 'propagate_cancel'),
+    ('stock.rule', 'stock_rule', 'propagate', 'propagate_cancel'),
+    ('stock.scrap', 'stock_scrap', 'date_expected', 'date_done'),
+]
+
+_column_renames = {
+    'stock_inventory': [
+        ('category_id', None),
+        ('filter', None),
+        ('location_id', None),
+        ('lot_id', None),
+        ('package_id', None),
+        ('partner_id', None),
+        ('product_id', None),
+    ],
+}
+
+_xmlid_renames = [
+    # mail.template
+    ('delivery.mail_template_data_delivery_confirmation', 'stock.mail_template_data_delivery_confirmation'),
+    # ir.model.access
+    ('stock.access_stock_fixed_putaway_strat', 'stock.access_stock_putaway_manager'),
+    ('stock.access_stock_fixed_putaway_user', 'stock.access_stock_putaway_all'),
+    ('stock.access_stock_forecast_user', 'stock.access_report_stock_quantity'),
+    # ir.actions.report
+    ('stock_zebra.label_barcode_product_product', 'stock.label_barcode_product_product'),
```

Comment:
These lines are not needed, as the module merge already performs this:

https://github.com/OCA/OpenUpgrade/blob/80a6e24d63ea80c61746d11b4124a5217aee3ccc/odoo/addons/openupgrade_records/lib/apriori.py#L30